ci/config/molecule.yaml

```diff
@@ -0,0 +1,12 @@
+- job:
```

Comment:
I'd rather use the role name instead of the job name. It would avoid some typos imho.

Reply:
Sorry, I don't get this. This YAML will be used for being merged to what the j2 generates based on the name of the job.

```
{% for role_name in role_names %}
- job:
    name: cifmw-molecule-{{ role_name }}
    parent: cifmw-molecule-base
    vars:
      TEST_RUN: {{ role_name }}
    files:
      - ^ansible-requirements.txt
      - ^molecule-requirements.txt
      - ^ci_framework/roles/{{ role_name }}/(?!meta|README).*
      - ^ci/playbooks/molecule.*
{% endfor %}
```

Reply:
oh, like that. OK, makes sense then.